### DISCUSSION TITLE
EF Nonresidential Maintenance and Repair(230301) or Facilities Support Services (561200)

### DISCUSSION THREAD

**Discussion by jvgregg** [USEPA/supply-chain-factors#36] [General]
Hi everyone, 
I am looking for the appropriate emission factor for various building maintenance and repair services such as: Elevator, HVAC, UPS, generators and general building repairs. 

I came across two categories that would apply: 
561200 - Facilities support services, and
230301 Non residential maintenance and repair

230301 does not have a NAICS description and 561200 it sates: "This industry comprises establishments primarily engaged in providing operating staff to perform a combination of support services within a client's facilities.  Establishments in this industry typically provide a combination of services, such as janitorial, maintenance, etc...)

Any thoughts on which one of these would be a better fit? 

Thanks!

**Comment by WesIngwersen** [comment]
Good question.
The construction sectors in USEEIO and BEA IO tables (23*) do not match NAICS industries exactly
I would probably suggest 230301 for more specialized building repair work; 561200 would include more general maintenance like cleaning.
